### PR TITLE
Added Support for Pipelines, from version 5.x of ElasticSearch

### PR DESCRIPTION
**Pull request by mohit-soral**
2 Changes : 
1. Updated elasticsearch to 5.0.0 in fluent-plugin-elasticsearch.gemspec
2.i. Added pipeline conf parameter
2.ii. If value for pipeline is configured, adding it to the meta-data

- [ ] tests added
- [x] tests passing
- [x] README updated (if needed)
- [x] README Table of Contents updated (if needed)
- [x] History.md and `version` in gemspec are untouched
- [ ] backward compatible
- [ ] feature works in `elasticsearch_dynamic` (not required but recommended)

### CODE REVIEW

**Comment by mohit-soral** [comment]
Checked options in the PULL REQUEST TEMPLATE

**Comment by pitr** [comment]
Could you add docs for this key please. Looks good otherwise

**Comment by mohit-soral** [comment]
@pitr : Sorry this got delayed from my end. I completely lost track of this thread. Let me know if anything else is required from my end to close this

File: lib/fluent/plugin/out_elasticsearch.rb [hunk 1]
```diff
@@ -324,6 +325,10 @@ def write_objects(tag, chunk)
       meta.clear
       meta["_index".freeze] = target_index
       meta["_type".freeze] = target_type
+      
+      if @pipeline
```
Comment (cosmo0920) [inline lib/fluent/plugin/out_elasticsearch.rb]: Any chance to add test for this?
Or #266 solve this?

**Comment by cosmo0920** [comment]
Closing this in favor of https://github.com/uken/fluent-plugin-elasticsearch/pull/266.

**Pull request closed without merging**